src/types/getters.ts

```diff
@@ -52,12 +52,12 @@ export function getWellKnownSymbolPropertyOfType(
 	for (const prop of type.getProperties()) {
 		if (!prop.name.startsWith(prefix)) continue;
 
+		const declaration = prop.valueDeclaration ?? prop.getDeclarations()![0];
+
 		const globalSymbol = typeChecker.getApparentType(
 			typeChecker.getTypeAtLocation(
-				(
-					(prop.valueDeclaration as ts.NamedDeclaration)
-						.name as ts.ComputedPropertyName
-				).expression
+				((declaration as ts.NamedDeclaration).name as ts.ComputedPropertyName)
```

Comment:
Do we know for sure that `declaration` is always a `NamedDeclaration` and it's name is always a `ComputedPropertyName`?

If not, we should probably add this above:

```ts
if (!isNamedDeclaration(declaration) || declaration.name === undefined || !ts.isComputedPropertyName(declaration.name)) {
	continue;
}
```

`isNamedDeclaration` doesn't currently exists though. We should create it.